docs/pelion_ready/device-management-test.md

```diff
@@ -55,31 +12,32 @@ Device Management tests rely on the Python SDK to test the end-to-end solution.
 
 <span class="notes">**Note:** The Python SDK requires Python 2.7.10+ or Python 3.4.3+, built with SSL support.</span>
 
-## Testing setup
+
+## Test setup
 
 1. Import an example application for Device Management that contains the corresponding configuration for your target.
 
-   Please refer to the following [application example](https://github.com/ARMmbed/pelion-ready-example). It demonstrates how to connect to the Device Management service, register resources and get ready to receive a firmware update.
+   The [application example](https://github.com/ARMmbed/pelion-ready-example) demonstrates how to connect to the Device Management service, register Resources and get ready to receive a firmware update.
 
-   Also, there are board-specific applications that focus on providing more elaborate hardware features with Mbed OS and Device Management. These are available in the Pelion [quick start](https://cloud.mbed.com/quick-start).
+   There are also [board-specific applications](https://cloud.mbed.com/quick-start) that focus on providing more elaborate hardware features with Mbed OS and Device Management.
 
-1. Set a global `mbed config` variable `CLOUD_SDK_API_KEY` on the host machine valid for the account your device will connect to. For example:
+1. Set the global `mbed config` variable `CLOUD_SDK_API_KEY` on the host machine. It must be valid for the account your device will connect to. For example:
 
     ```
     $ mbed config -G CLOUD_SDK_API_KEY <API_KEY>
     ```
 
-    For instructions on how to generate an API key, please [see the documentation](https://cloud.mbed.com/docs/current/integrate-web-app/api-keys.html#generating-an-api-key).
+    See [instructions on how to generate an API key](https://www.pelion.com/docs/device-management/current/integrate-web-app/api-keys.html#generating-an-api-key).
```

Comment:
Should we change links in this, or do it all at once in its own PR?

Reply:
Didn't I just change them? I may have missed some.